static/src/javascripts/projects/common/modules/cmp-ui/index.js

```diff
@@ -31,6 +33,7 @@ export const init = (forceModal: boolean) => {
                 "'Guardian Text Sans Web', Helvetica Neue, Helvetica, Arial, Lucida Grande, sans-serif",
         },
         forceModal,
+        variant: isInVariantSynchronous(commercialCmpCopy, 'variant') ? 'commercialCmpCopy-variant' : 'commercialCmpCopy-control',
```

Comment:
I just realised this'll report all user's who aren't in the `variant` as being in the `control`, including those users who are not even in the test, we should only pass this `variant` prop if they're in the `variant`/`control` of  the `commercialCmpCopy` a/b test